hkmc2/shared/src/main/scala/hkmc2/syntax/ParseRule.scala

```diff
@@ -189,6 +191,24 @@ object ParseRule:
   val prefixRules: ParseRule[Tree] = ParseRule("start of statement", omitAltsStr = true)(
     letLike(`let`),
     letLike(`set`),
+    
+    Kw(`handle`):
+      ParseRule("'handle' binding keyword"):
+        Expr(
+          ParseRule("'handle' binding head"):
+            Kw(`=`):
+              ParseRule("'handle' binding equals sign"):
+                Expr(
+                  ParseRule("'handle' binding class name"):
+                    typeDeclTemplateThen(
+                      Kw(`in`):
+                        ParseRule(s"'handle' binding `in` clause")(
+                          exprOrBlk(ParseRule(s"'handle' binding body")(End(())))((body, _: Unit) => S(body))
+                      *),
```

Comment:
It seems this line is mis-indented.

Reply:
oh it seems just the last comma is misindented, will be fixed when I figure out the other change I'm working on

Reply:
Oh then the problem exists in `letLike` as well which I copy-pasted from. I'll fix that as well

Reply:
Probably. Feel free to fix it there as well! 😉